scripts/test-django-mt.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+pyenv global "${PYTHON_VERSION}"
+echo "${PATH}"
+pyenv global "${PYTHON_VERSION}"
+echo "Python version: $(python --version)"
+pip install -r /build/requirements/test-requirements.txt 
```

Comment:
BTW, looks like this is the same as the [make dev-dependencies](https://github.com/citusdata/django-multitenant/blob/citus_matrix/Makefile#L18-L19) task. Maybe we should call that here, so that if the command changes we need to change it in only one place.

Reply:
Fixed as you suggested